src/components/Button/ButtonStyle.ts

```diff
@@ -2,11 +2,14 @@ import { css } from "lit";
 
 export default css`
   .button-container {
-    border: color: transparent;
+    border-color: transparent;
     font-weight: 500;
     text-decoration: none;
     border-style: none;
     font-family: inherit;
+    display: flex;
+    gap: 0.50rem;
+    align-items: center;
```

Comment:
missing `justify-content: center`
Otherwise it will always stick to the left

Add also a `min-height: 2.5rem`
Button with only icon is a bit tinier than with text. It will normalize button's height